cmd/tether/tether_linux.go

```diff
@@ -117,7 +117,7 @@ func (t *osopsLinux) setup() error {
 		log.Error(detail)
 		return errors.New(detail)
 	}
-	log.SetOutput(io.MultiWriter(out, os.Stdout))
+	log.SetOutput(io.MultiWriter(os.Stdout, out))
```

Comment:
Apparently io.MultiWriter quits on the first stream that has an error. In my testing on my linux box, writing to ttySx was failing, but I still wanted to see the log output on stdout.